app/db.py

```diff
@@ -4,14 +4,19 @@
 import psycopg2
 
 
-def connect(db, user, host):
+def connect():
     """ connect to the database """
-    if os.environ.get('test'):
+    if os.environ.get('ENV_MODE') == 'test':
         db = 'testsnp'
+    else:
+        db = 'snp'
 
-    return psycopg2.connect(database=db, user=user, host=host)
+    return psycopg2.connect(database=db,
+        user=os.environ.get('DB_USER'),
+        host=os.environ.get('DB_HOST'),
+        port=os.environ.get('DB_PORT'))
```

Comment:
It's a small thing, but I think the environment variables will help in testing back and forth between aws and laptops.